Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2856,29 +2982,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "118":
-    id: "118"
-    taskid: 93c3a0af-b2c4-4721-8984-0d86b28ed61b
-    type: playbook
+  "130":
+    id: "130"
+    taskid: d7e54556-33f6-40d2-85c3-9497e480c8cd
+    type: regular
     task:
-      id: 93c3a0af-b2c4-4721-8984-0d86b28ed61b
+      id: d7e54556-33f6-40d2-85c3-9497e480c8cd
       version: -1
-      name: Cortex ASM - Remediation
-      playbookName: Cortex ASM - Remediation
-      type: playbook
+      name: Set playbook stage grid field (decision)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```